test/adaptive_load/scoring_function_test.cc

```diff
@@ -0,0 +1,231 @@
+#include "external/envoy/source/common/config/utility.h"
+
+#include "adaptive_load/scoring_function_impl.h"
+#include "gtest/gtest.h"
+
+namespace Nighthawk {
+
+namespace {
```

Comment:
If using an anonymous namespace here is what we want, we may want to file an issue to retrofit it everwhere for consistency.

Reply:
Filed #431